README.md

```diff
@@ -1,2 +1,158 @@
-# EventPage
-A page made to advertise a Catholic concert
+<a name="readme-top"></a>
+
+<div align="center">
+   <img src="images/logo-big.png" alt="logo" width="140"  height="auto" />
+  <br/>
+  <h1><b>Catholic Music Concert - United by the same Faith</b></h1>
+
+</div>
+
+# 📗 Table of Contents 
+
+- [📖 About the Concert](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Walkthrough to the Concert Page](#walkthrough)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 [Catholic Music Concert] <a name="about-project"></a>
+
+**[The Concert Page]** is a development with all the information about the **"CMC - Catholic Music Concert 2023"**
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com/">HTML</a></li>
+    <li><a href="https://www.w3.org/standards/">CSS</a></li>
+  </ul>
+</details>
+  
+### Key Features <a name="key-features"></a>
+
+**Enlist skill sets and tech stack**
+- Responsive design
+- Mobile First methodology
+- Walkthrough to the [Concert Page](https://www.loom.com/)
+
+### Walktrought Concert Page <a name="walktrought"></a>
+
+**Catholic Music Concert - Microverse - Capstone Project - Watch Video**
+
+[<img style="max-width:300px;" src="https://cdn.loom.com/sessions/thumbnails/165bf7bc353a424195f804718c734c7f-with-play.gif">](https://www.loom.com/share/165bf7bc353a424195f804718c734c7f)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
as required your video maximum length should be 5minute please rerecord your video and present your project in 5minutes 💯